js/src/launchResponse.ts

```diff
@@ -4,5 +4,8 @@
 export type LaunchResponse = {
     container: HTMLDivElement;
     sessionId: string;
+    readerReadyDuration: number;
+    launchDuration: number;
+    gcmCorrelationId: string;
     charactersProcessed: number;
```

Comment:
please keep this order for the properties (exactly as internal sdk):

container
charactersProcessed
readerReadyDuration
launchDuration
gcmCorrelationId
sessionId